ketcher-autotests/tests/test-data/KET/CHEM-Templates/11 - (R3,R4,R5)-expected.ket

```diff
@@ -16,8 +16,8 @@
         "type": "monomer",
         "id": "1",
         "position": {
-            "x": 3.1733333333333333,
-            "y": -2.219999999999999
+            "x": 11.9,
```

Comment:
question about coordinates: is it normal that they began to be rounded to tenths of x?

Reply:
Why do you think _they began to be rounded_?

Reply:
Because I haven't seen this before. Is this normal for the coordinates? Or were some fixes made?

Reply:
Javascript automatically sets 11.90000000000 to 11.9.